go/enclave/services/README.md

```diff
@@ -0,0 +1,5 @@
+### The Service package
```

Comment:
In golang it's common practice to avoid generic packages like services.
Instead, for example, nodetype could work, where the `sequencer` and the `validator` are implementations of the `node_type.go` interface. 

If another name is prefered that's good too, it's just going in the direction of this package is a unit of work and not a collection of units. Hope that makes sense :)